itl/plugins-contrib.d/network-components.conf

```diff
@@ -604,7 +604,7 @@ object CheckCommand "nwc_health" {
 			description = "The role of this device in a hsrp group (active/standby/listen)"
 		}
 		"--report" = {
-			set_if = "$nwc_health_report$"
+			value = "$nwc_health_report$"
```

Comment:
Would be cool if you can add that to both, the description field and also the docs :)